### ISSUE TITLE
bug: blue progress bar stalls and never completes

### ISSUE THREAD

**Issue by ericberlow** [ericberlow/openmappr#7]
https://cl.ly/342k3A0v120y

here is the project
http://openmappr.swall.org/projects/5b4f5cbddedeac0013545615

**Comment by ericberlow** [comment]
@mandeepm91  - kaustuv mentioned that bimal would be good to ask about the status/progress bar

**Comment by mandeepm91** [comment]
@ericberlow I had opened an issue with the main repo last week. Seems like both are related
I am referring to this issue: https://github.com/VibrantData/openmappr/issues/16

The problem was that our code was referencing an non existing file which was stalling the blue progress bar. I have fixed it now and deployed the changes to the server. Let me know if all looks good. Bimal is on vacation for now. I will connect with him once he is back to double check.
